Don't use svnversion or git describe since we're not using a svn wc or
git clone and since the svnversion or git commands might not exist.
--- libelftc/make-toolchain-version.orig	2016-02-16 16:58:35.000000000 -0600
+++ libelftc/make-toolchain-version	2021-07-02 13:06:53.000000000 -0500
@@ -64,6 +64,8 @@
 curdir=`pwd`
 cd ${top} || usage "ERROR: Cannot change directory to \"${top}\"."
 
+versionstring=
+if false; then
 if [ -d CVS ]; then		# Look for CVS (NetBSD).
     versionstring=" cvs:unknown"
 else				# Try git (DragonFlyBSD).
@@ -82,6 +84,7 @@
     echo "ERROR: cannot determine a revision number." 1>&2
     exit 1
 fi
+fi
 
 cd ${curdir} || usage "Cannot change back to ${curdir}."
 
